Portal/View/UIKit/PortalCollectionView.swift

```diff
@@ -76,6 +80,15 @@ public class PortalCollectionView<
         item.onTap |> { mailbox.dispatch(message: $0) }
     }
     
+    public func configRefresh(properties: RefreshProperties<ActionType>, tintColor: Color) {
```

Comment:
I would extract this into an extension in `PullToRefreshable` protocol. Also I don't think `RefreshController` is necessary you can do all the configuration in the following method and use a `MessageDispatcher` to send the required message when the user pulls to refresh.

```swift
internal protocol PullToRefreshable {

    associatedtype MessageType

    var scrollView: UIScrollView { get }

    var mailbox: Mailbox<MessageType>

    func configure(pullToRefresh properties: RefreshProperties<ActionType>, tintColor: Color)

}

extension PullToRefreshable where Self == UIScrollView {

    var scrollView: UIScrollView { 
        return self 
    } 

}

extension PullToRefreshable {

    func configure(pullToRefresh properties: RefreshProperties<ActionType>, tintColor: Color) {
        let refreshControl = UIRefreshControl()
        refreshControl.tintColor = tintColor.asUIColor
        refreshControl.attributedTitle = properties.title
        
        switch properties.state {
            
        case .searching:
            scrollView.contentOffset = CGPoint(x:0, y: -refreshControl.frame.size.height)
            refreshControl.beginRefreshing()
            
        case .idle(let message):
            let dispatcher = MessageDispatcher(mailbox: mailbox, message: message)
            self.register(dispatcher: dispatcher)
            refreshControl.dispatch(message: message, for: .valueChanged, with: mailbox) }

        }

        scrollView.refreshControl = refreshControl
    }

}
```